# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../kwlan/stdlib_fix.patch
# Copyright (C) 2006 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---
#
# Patch by Christian Wiese. 'Thanks bro' -Nagy Karoly Gabriel

diff -ruN kwlan-0.5.3-orig/src/main.cpp kwlan-0.5.3/src/main.cpp
--- kwlan-0.5.3-orig/src/main.cpp	2006-08-27 19:54:53.000000000 +0300
+++ kwlan-0.5.3/src/main.cpp	2006-08-29 20:00:06.000000000 +0300
@@ -1,5 +1,5 @@
 
-
+#include <stdlib.h>
 #include "kwlanmainwin.h"
 #include <kuniqueapplication.h>
 #include <dcopclient.h>
diff -ruN kwlan-0.5.3-orig/src/suprocessbase.h kwlan-0.5.3/src/suprocessbase.h
--- kwlan-0.5.3-orig/src/suprocessbase.h	2006-07-11 00:22:48.000000000 +0300
+++ kwlan-0.5.3/src/suprocessbase.h	2006-08-29 19:47:10.000000000 +0300
@@ -21,7 +21,7 @@
  *  SuProcessBase is Copyright (c) 2006 by Valentine Sinitsyn <e_val@inbox.ru>. Distributed under terms of LGPL.
  */
 
-#define DEFAULT_SUPER_USER_COMMAND "sudo"
+//#define DEFAULT_SUPER_USER_COMMAND "sudo"
 
 class SuProcessBase: public QObject {
   Q_OBJECT
